Brainfuck/spark/README.md

```diff
@@ -0,0 +1,5 @@
+TLDR; usage
+
+- Follow the instructions for downloading and setting up Alire https://alire.ada.dev
+- `alr gnatprove` will run the proofs.
+- `alr run --args=../_tests/helloworld.bf` will run the example program.
```

Comment:
Last thing, could you move these comments into programy.yaml in the `notes` objects? https://github.com/eatonphil/tinyprograms/blob/main/Brainfuck/freepascal/program.yaml#L7

And we can get rid of the README.

Reply:
Those notes get rendered into the UI: https://tinyprograms.org/Brainfuck/freepascal.html